### PR TITLE
47 auto update release model info

### LINKED ISSUE #47: Auto submit a PR when update the release
So far, after merging a PR, if the release has been updated, a new branch will be pushed with the updated `model_info.json`, but we still need an admin to manually create a PR for the branch.

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #47 .

### Description
This PR has the following updates:
1. for `update-model-info` workflow, change to trigger only on `workflow_dispatch`. This change is used to avoid raising 403 error when a PR from forked repo is merged. In the future, when a PR is merged, our admin can run this workflow, and then a changed bundle will be updated into the release.
2. auto add a pull request which contains the changed `model_info.json`.
3. remove mypy check, it is because each bundle in this repo is independent, do mypy check will consider them together, which may raise unnecessary errors.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json`.
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
I tested this PR with my forked repo:
https://github.com/yiheng-wang-nv/model-zoo/runs/7739260622?check_suite_focus=true#step:3:94

The auto submitted PR is like the following (Hi @wyli , do you know why pre-commit workflow does not run in this PR?):
https://github.com/yiheng-wang-nv/model-zoo/pull/3

**Comment by wyli** [comment]
It could also be an automated daily job on the Dev branch to check for model updates? In case the admin forget to trigger the workflow.

**Comment by yiheng-wang-nv** [comment]
Hi @wyli , thanks for the suggestion. Yes, we can also set a daily job, and check if the bundle (with version) is existing in release. The missing bundles will be uploaded, with a PR that contains the changed `model_Info.json`. Hi @Nic-Ma , what do you think?

**Comment by Nic-Ma** [comment]
If we changed many bundles in many PRs within one day, please ensure that no conflicts or errors in the `model_info.json`.
Otherwise, I feel fine about the idea.

Thanks.

**Comment by yiheng-wang-nv** [comment]
Update: now the workflow is scheduled everyday at 10:00 am (UTC time), and can be manually triggered.

File: ci/utils.py [hunk 1]
```diff
@@ -98,6 +98,14 @@ def push_new_model_info_branch(model_info_path: str):
     call_status = subprocess.run(full_cmd, shell=True)
     call_status.check_returncode()
 
+    return branch_name
+
+
+def create_pull_request(branch_name: str):
+    create_command = f"gh pr create --fill --title 'auto update model_info' --base dev --head {branch_name}"
```
Comment (wyli) [inline ci/utils.py]: this hard coded `title` `auto update model_info` could be an option to `create_pull_request`?

**Review by wyli — APPROVED** [review]

**Pull request merged**